libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/defaults.parm

```diff
@@ -0,0 +1,13 @@
+# Serial3 default to GPS
```

Comment:
my mistake....SERIAL3 already defaults to GPS....so not needed.....but need to fix UART order in hwdef so that UART7 is third and UART6 is forth, not the other way around...